MerlinAU.sh

```diff
@@ -790,37 +790,31 @@ Get_Custom_Setting()
     fi
 }
 
-##------------------------------------------##
-## Modified by ExtremeFiretop [2024-Apr-26] ##
-##------------------------------------------##
+##----------------------------------------##
+## Modified by Martinski W. [2024-Apr-30] ##
+##----------------------------------------##
 _GetAllNodeSettings_()
 {
-    if [ $# -lt 2 ] || [ -z "$1" ] || [ -z "$2" ]; then return 1; fi
+    if [ $# -lt 2 ] || [ -z "$1" ] || [ -z "$2" ]
+    then echo "**ERROR**" ; return 1; fi
 
-    local node_mac_address="$1"  pattern
-    local setting_value  setting_part="$2"  matched_lines
-    local default_value="TBD"
+    ## Node Setting KEY="Node_{MACaddress}_{keySuffix}" ##
```

Comment:
Approved and tested with success :)
Thank you for the additional cleanup! Your touch is always appreciated